library/src/main/java/com/novoda/downloadmanager/lib/DownloadReadyChecker.java

```diff
@@ -37,6 +37,7 @@ private boolean isDownloadManagerReadyToDownload(FileDownloadInfo downloadInfo)
         }
         switch (downloadInfo.getStatus()) {
             case 0: // status hasn't been initialized yet, this is a new download
+            case DownloadStatus.SUCCESS:
```

Comment:
it should be denied by this method though ^ 
we need to query for all downloads in order to update the notification